specifyweb/frontend/js_src/lib/components/AttachmentsBulkImport/ViewAttachmentFiles.tsx

```diff
@@ -0,0 +1,214 @@
+            }
+            onClick={handleDisambiguate}
+          >
+            {resolvedRecord?.type === 'matched' ? (
+              <Link.NewTab
+                href={getResourceViewUrl(baseTableName!, resolvedRecord.id)}
+              >
+                {file.parsedName?.toString()}
+              </Link.NewTab>
+            ) : (
+              resolvedRecord !== undefined && (
+                <div>
+                  {keyLocalizationMapAttachment[resolvedRecord.reason] ?? ''}
+                </div>
+              )
+            )}
+          </div>,
+        ],
+
+        attachmentId,
+        canDisambiguate: typeof handleDisambiguate === 'function',
+        isNativeError: resolvedRecord?.type === 'invalid',
+        isRuntimeError,
+      } as const;
+    }
+  );
+
+export function ViewAttachmentFiles({
+  uploadableFiles,
+  baseTableName,
+  uploadSpec,
+  onDisambiguation: handleDisambiguation,
+  onFilesDropped: handleFilesDropped,
+}: {
+  readonly uploadableFiles: RA<PartialUploadableFileSpec>;
+  readonly baseTableName: keyof Tables | undefined;
+  readonly uploadSpec: PartialAttachmentUploadSpec;
+  readonly onDisambiguation:
+    | ((
+        disambiguatedId: number,
+        indexToDisambiguate: number,
+        multiple: boolean
+      ) => void)
+    | undefined;
+  readonly onFilesDropped?: (file: FileList) => void;
+}): JSX.Element | null {
+  const [disambiguationIndex, setDisambiguationIndex] = React.useState<
+    number | undefined
+  >(undefined);
+
+  const data = React.useMemo(
+    () =>
+      resolveAttachmentDatasetData(
+        uploadableFiles,
+        setDisambiguationIndex,
+        baseTableName
+      ),
+    [uploadableFiles, setDisambiguationIndex, baseTableName]
+  );
+  const headers = React.useMemo(
+    () => ({
+      selectedFileName: commonText.selectedFileName(),
+      fileSize: attachmentsText.fileSize(),
+      record: (
+        <div className="flex min-w-fit items-center gap-2">
+          {baseTableName === undefined ? (
+            formsText.record()
+          ) : (
+            <>
+              <TableIcon label name={baseTableName} />
+              {uploadSpec.staticPathKey === undefined
+                ? ''
+                : defined(
+                    strictGetModel(baseTableName).getField(
+                      staticAttachmentImportPaths[uploadSpec.staticPathKey].path
+                    )
+                  ).label}
+            </>
+          )}
+        </div>
+      ),
+      status: attachmentsText.status(),
+      attachmentId: attachmentsText.attachmentID(),
+    }),
+    [uploadSpec.staticPathKey]
+  );
+
+  const fileDropDivRef = React.useRef<HTMLDivElement>(null);
+  const { isDragging, ...restCallbacks } = useDragDropFiles(
+    handleFilesDropped,
+    fileDropDivRef
+  );
+
+  const headerElements = React.useMemo(
+    () => Object.values(headers).map((label) => <>{label}</>),
+    [headers]
+  );
+
+  return data.length === 0 ? null : (
```

Comment:
While I suggested that the table shouldn't be shown when there is nothing to show as it seemed weird to have a table with header but without any rows, the page seems quite empty now (almost like some UI was supposed to be loaded in the middle, but didn't because of a bug). And it's not clear to user what is expected of them next

![image](https://github.com/specify/specify7/assets/40512816/aba2b8e7-e7ea-4945-a4af-c2b30922009f)

yes, there is "Choose a file or drag it here", but it's not very prominent - since that it the action expected of user next, it should be much more visible.

I would suggest moving that small "Choose a file or drag it here" button to be in the main content area (exactly how it is in the workbench import).
AND if you already imported some, then either display that to the right of the table (since you have a lot of unused space there), OR below the table (since that would be intuitive - any additional attachments you drag and drop will appear below the table). You will still support drag and drop into the table, so the user won't be required to scroll to the bottom of the table to import more attachments.

How does that sound?

That would also make the top header less busy

Reply:
**Conclusion:**
* Allow users to drag & drop files into the blank space
* Add back the background
* Add instructions that explain what files can be batch uploaded
   * "You can select files that are named to match existing records:"
      * Catalog Number - Collection Object
      * Taxon - Full Name
      * Station Field Number – Collecting Event
      * (just to give an idea of how this works)
   * "Drag and drop attachments here to get started" after this
 * Link to Discourse for expanded documentation

Reply:
Fixed